modules/nw-ovn-kubernetes-matrix.adoc

```diff
@@ -11,60 +11,45 @@
 .Default CNI network plugin feature comparison
 [cols="50%,25%,25%",options="header"]
 |===
-ifeval::["{context}" == "about-ovn-kubernetes"]
-|Feature|OVN-Kubernetes|OpenShift SDN
 
-|Egress IPs|Supported|Supported
-
-|Egress firewall ^[1]^|Supported|Supported
-
-|Egress router|Supported ^[2]^|Supported
-
-|Hybrid networking|Supported|Not supported
-
-|IPsec encryption for intra-cluster communication|Supported|Not supported
-
-|IPv6|Supported ^[3]^ ^[4]^|Not supported
-
-|Kubernetes network policy|Supported|Supported
-
-|Kubernetes network policy logs|Supported|Not supported
-
-|Hardware offloading|Supported|Not supported
-
-|Multicast|Supported|Supported
-endif::[]
-ifeval::["{context}" == "about-openshift-sdn"]
 |Feature|OpenShift SDN|OVN-Kubernetes
 
 |Egress IPs|Supported|Supported
 
-|Egress firewall ^[1]^|Supported|Supported
+|Egress firewall|Supported|Supported ^[1]^
 
 |Egress router|Supported|Supported ^[2]^
 
 |Hybrid networking|Not supported|Supported
 
-|IPsec encryption|Not supported|Supported
+|IPsec encryption for intra-cluster communication|Not supported|Supported
+
+|IPv4 single-stack|Supported|Supported
+
+|IPv6 single-stack|Not supported|Supported ^[3]^
+
+|IPv4/IPv6 dual-stack|Not Supported|Supported ^[4]^
 
-|IPv6|Not supported|Supported ^[3]^ ^[4]^
+|IPv6/IPv4 dual-stack|Not supported|Supported ^[5]^
```

Comment:
@dfitzmau do you mean single stack here or whats the difference to line 31? trying to understand why IBM Z isn't supported here but IBM Power is. 
```suggestion
|IPv6/IPv4 single-stack|Not supported|Supported ^[5]^
```

Reply:
The primary (first) IP family cannot change, so the dual-stacks are different.

IPv6/IPv4 is the only dual-stack that can convert to IPv6 single-stack at runtime.

Reply:
Dual-Stack is supported for Power

Reply:
@mtarsel @sajauddin please confirm.